runner/Dockerfile

```diff
@@ -3,9 +3,15 @@ FROM ubuntu:18.04
 ARG RUNNER_VERSION
 ARG DOCKER_VERSION
 
-RUN apt update \
-  && apt install sudo curl ca-certificates -y --no-install-recommends \
-  && curl -L -o docker.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz \
+RUN apt update -y \
+  && apt install -y software-properties-common \
+  && add-apt-repository -y ppa:git-core/ppa \
+  && apt update -y \
+  && apt install sudo curl ca-certificates unzip git -y --no-install-recommends 
+  && apt clean \
```

Comment:
According to Docker's best practice, this is not necessary now ;)
> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.


https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

Reply:
Thanks, I wasn't 100% sure about it. I did keep the `rm -rf /var/lib/apt/lists/*` though (saves about 30 Mb on the final image)

Reply:
Yeah, I think `rm -rf /var/lib/apt/lists/*` is still needed today ;)